setup.cfg

```diff
@@ -88,4 +88,4 @@ exclude_lines =
 [options.entry_points]
 console_scripts =
```

Comment:
In the end, we want to remove `setup.cfg`

You want to remove this and then make sure the tests pass.

Reply:
also I will suggest adding some reference links in the pr description on why we're switching from setup.cfg to pyproject.toml

Reply:
on a side note, we also want to perhaps remove setup.py- these files are outdated and python community has better ways for packaging if you could research since a lot of time has passed since when I researched about it